desktop/libs/indexer/src/indexer/templates/importer.mako

```diff
@@ -554,6 +554,11 @@ ${ commonheader(_("Importer"), "indexer", user, request, "60px") | n,unicode }
               <label for="dialectType" class="control-label "><div>${ _('Dialect') }</div>
                 <select  id="dialectType" data-bind="selectize: $parent.createWizard.source.interpreters, value: $parent.createWizard.source.interpreter, optionsText: 'name', optionsValue: 'type'"></select>
               </label>
+              <div data-bind="visible: dialect() == 'phoenix'">
```

Comment:
for the create statement (in phoenix) we need a primary key so the list of col added in a drop-down.

Reply:
There is already the concept of PKs with Hive/Kudu format, not way to refactor to use and avoid more duplication?